nestkernel/nest.h

```diff
@@ -121,6 +121,8 @@ const index invalid_index = std::numeric_limits< index >::max();
 typedef unsigned char synindex;
 const synindex invalid_synindex = std::numeric_limits< synindex >::max();
 
+#define UNLABELED_CONNECTION -1
```

Comment:
@tammoippen Is it really necessary to use a C makro here instead of a C++ constant? And is `nest.h` really the best place to define the constant? Wouldn't `connection_label.h` be a better place, or would that require too many includes all across the code?